pontoon/base/static/js/translate.js

```diff
@@ -3032,11 +3052,14 @@ window.onpopstate = function(e) {
   }
 };
 
+var preferredLocales = $('#server').data('preferred-locales');
+
 Pontoon.user = {
   email: $('#server').data('email') || '',
   name: $('#server').data('name') || '',
   forceSuggestions: $('#server').data('force-suggestions') === 'True' ? true : false,
-  manager: $('#server').data('manager')
+  manager: $('#server').data('manager'),
+  preferredLocales: preferredLocales ? preferredLocales : {}
```

Comment:
Would `$('#server').data('preferred-locales') || {}` also work instead of this and line 3055?